datatypes/src/raster/operations/map_indexed_elements.rs

```diff
@@ -346,7 +346,7 @@ where
             .map(|(lin_idx, i)| map_fn(lin_idx, i))
             .collect_into_vec(out_data.as_mut());
 
-        Grid::new(shape, out_data).expect("Grid creation with shape failed before.")
+        Grid::new(shape, out_data).expect("Grid should have been created.")
```

Comment:
not only should it be created. A masled  grid with the same shape etc. already existed. if it worked before  it must work here.

Reply:
0bc74c57b981a9d1509efdf2b3c5b412f68e404f